master branch won't build for me

I get the following when trying to build the master branch:

```*:cwd games
*:midas jotto
JOTTO HACK
JOTTO HACK
Constants area inclusive
From    To
54362   54633
Run time = 0.03
1621 Symbols including initial ones (59% used)

:KILL
*:job jotto
!
*:load jotto bin
*erase0$b   beg$g
$1B; ERASE0>>MOVE R,54456   utopen/   656470   $1'   dsk$
b/   0   $1'jotto$
c/   0   $1'dict$
beg7$g

6627.
IT IS NOW  5:53:18 PM EDT, THURSDAY, AUG 2,2018
words
$1B; ERASE0>>MOVE R,54456
The last command timed out.
Makefile:50: recipe for target 'out/klh10/rp0.dsk' failed
make: *** [out/klh10/rp0.dsk] Error 1
eswenson@eswenson-ThinkPad-W520:~/ITS/ws/its$```

It seems that darned "IT IS NOW..." message breaks the expect (ations).

--- a/build/misc.tcl
+++ b/build/misc.tcl
@@ -466,7 +466,7 @@ type "c/\0331'dict\033\r"
 # Run the dictionary loader, skipping its filename setup code.
 type "beg7\033g"
 # Dump out TS JOTTO including the dictionary.
-respond ". words" ":pdump sys1;ts jotto\r"
+respond "words" ":pdump sys1;ts jotto\r"
 respond "*" ":kill\r"
 
 # ngame

